src/UI/Component/Glyph/Factory.php

```diff
@@ -699,4 +699,38 @@ public function angry($action = null);
 	 * @return \ILIAS\UI\Component\Glyph\Glyph
 	 */
 	public function attachment($action = null);
+
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *      The Apply Glyph is used to indicate the possibilty of applying changes which the user has made
+	 *      within a control, i.e. a filter.
+	 *   composition: >
+	 *      The Apply Glyph uses the glyphicon-ok.
+	 *   effect: >
+	 *       Upon clicking the page is reloaded immediately with the updated content the control affects. In case of
+	 *       a filter, it means that the entries in a table change in accordance with the filter values.
+	 *
+	 * featurewiki:
+	 *       - https://www.ilias.de/docu/goto.php?target=wiki_1357_Responsive_Table_Filters#ilPageTocA121
+	 *
+	 * rules:
+	 *   usage:
+	 *       1: >
+	 *          The Apply Glyph SHOULD not come without a Reset Glyph and vice versa.
+	 *       2: >
+	 *          If there are no changes to apply, the Apply Glyph MUST be deactivated (or not clickable).
+	 *   style:
+	 *       1: >
+	 *          The deactivated state of the Apply Glyph MUST be visually noticeable for the user, i.e. by greying out
+	 *          the Apply Glyph.
```

Comment:
We do not convey information only by color. I would like an aria-disabled state here or similar.

Reply:
So we need a new method like `withDisabled` here, right? This would affect the existing glyphs, too. Is this desired?

Reply:
Why is this different from the deactivate state of the color?

Reply:
It's not different. So we need the new method for the aria-disabled state and also the color state, right?

Reply:
oke, I thought from reading the initial PR, that you would handle the implicitly in the proposal somehow and do the same with the aria label. However thinking about it, that would be really best handled by making this explicit on the interface. For Buttons we call that "withUnavailableAction" (Desc: Get a button like this, but action should be unavailable atm. The button will still have an action afterwards, this might be usefull at some point where we want to reactivate the button client side), I think that would also fit here. Do you agree?

Reply:
The Glyphs have the new method now. Aria-disabled state and color state are implemented. 

But I did not found a way to make Glyphs not clickable. For Buttons, we obviously use "ilSubmitInactive disabled" for the Button class in the template. But I did not figure how exactly the "not clickable state" is achieved. Could you maybe explain it to me?

Reply:
It does, many thanks! If my understanding is correct, this is coming from the Bootstrap css. That's why I did not found it because I was focusing on the specific css/less file for the Buttons. But I would like to make an annotation to the example you are referring to: The Button template from Ilias 5.3 is already deprecated. Buttons are created with the <a>-tag there. In trunk, we use the <button>-tag. I took a closer look at the stuff with the disabled buttons and found out, that buttons created as <button> support the disabled attribute, while buttons created with <a>-tag don't support that (See [Disabled state](https://getbootstrap.com/docs/4.0/components/buttons/#disabled-state). It refers to Bootstrap 4, but also goes for Bootstrap 3). That's the reason why we need the a.btn.disabled class to handle the pointer-event in Ilias 5.3, like you mentioned in your last comment. Since Ilias 5.4, this is no longer required for KS-Buttons.

Back to the Glyphs. In this case, your example was very helpful, because unlike Buttons, Glyphs do not support the disabled attribute. So I added the pointer-event stuff to the css class.